Formula/ola.rb

```diff
@@ -31,8 +31,7 @@ class Ola < Formula
   end
 
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].bin/"python3"
-    protobuf_pth = Formula["protobuf@3.6"].opt_lib/"python#{xy}/site-packages/homebrew-protobuf.pth"
+    protobuf_pth = Formula["protobuf"].opt_prefix/Language::Python.site_packages("python3")/"homebrew-protobuf.pth"
```

Comment:
Actually, I'm a bit surprised this works, as there is no `homebrew-protobuf.pth` at `$HOMEBREW_PREFIX/opt/protobuf/lib/python3.9/site-packages`. There is a `protobuf-3.17.3-py3.9-nspkg.pth`, though.

Reply:
```suggestion
```
Let's just get rid of it; I don't think this does anything except install a broken symlink.